packages/core/src/hooks/useTokenList.ts

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from 'react'
+import axios from 'axios'
+import { TokenInfo } from '@uniswap/token-lists'
+import { ChainId } from '../constants'
+
+interface TokenList {
+  name: string
+  logoURI: string
+  tokens: TokenInfo[]
+}
+
+export function useTokenList(tokenListURI: string, chainId: ChainId = ChainId.Mainnet) {
```

Comment:
I propose default chainId to be taken from currently connected network, that would be sth like this:
```ts
export function useTokenList(tokenListURI: string, overrideChainId: ChainId = null) {
  const {chainId} = useEthers()
  if (!overrideChainId) {
     overrideChainId = chainId
  } 

....
          tokens: (tokens as TokenInfo[]).filter((token) => token.chainId === overrideChainId),
...
  }, [tokenListURI, overrideChainId])
...
```

Reply:
If chainId undefined/null - return empty list

Reply:
✅